pkg/controller/enterprisesearch/config.go

```diff
@@ -22,12 +19,20 @@ import (
 	"github.com/elastic/cloud-on-k8s/pkg/controller/common/volume"
 	"github.com/elastic/cloud-on-k8s/pkg/controller/enterprisesearch/name"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/rand"
 )
 
 const (
 	ESCertsPath     = "/mnt/elastic-internal/es-certs"
 	ConfigMountPath = "/mnt/elastic-internal/config"
 	ConfigFilename  = "enterprise-search.yml"
+
+	SecretSessionKey  = "secret_session_key"
+	EncryptionKeysKey = "secret_management.encryption_keys"
```

Comment:
there's a `Key` prefix to `EncryptionKeysKey ` but not `SecretSessionKey `. Would `setting` look a bit better?
```go
SecretSessionSetting  = "secret_session_key"
EncryptionKeysSetting = "secret_management.encryption_keys"
```